### PR TITLE
feat: adding toolbar and backbutton functionality in some activities.

### LINKED ISSUE #770: Help & feedback layout not having toolbar
**Actual Behavior**
Just white void layout comes up.

**Expected Behavior**
Should have FAQ and help instructions.

**Steps to reproduce it**
Simply implementing toolbar layout and back button functionality on it. 

**Screen-shot of the issue**
bug_report

**Would you like to work on the issue?**

_**Yes**_,I would like to work on this.

### PR DESCRIPTION
**Pull request by bravoo84**
Fixes #770 

Changes: added help_feedback_toolbar.xml and changed helpFeedback.java

APK for testing: Sending in next message

### CODE REVIEW

**Comment by bravoo84** [comment]
**gif of changes made:**
bug_update2

**Comment by bravoo84** [comment]
Sir this is my first PR in this project ,I will improve the quality from next time.

**Comment by bravoo84** [comment]
@CloudyPadmal please review my PR.

**Comment by bravoo84** [comment]
**fixes issue #770**

**Gif of the fixed issue**
bug_report3


**Apk of updated app**
[app-debug.zip](https://github.com/fossasia/pslab-android/files/1845671/app-debug.zip)

**Comment by bravoo84** [comment]
@CloudyPadmal please review my 2nd commit which under this same PR.

File: app/src/main/res/layout/sensor_main.xml [hunk 1]
```diff
@@ -3,8 +3,20 @@
...
+    <android.support.v7.widget.Toolbar
+        android:layout_width="match_parent"
+        android:layout_height="?attr/actionBarSize"
+        android:id="@+id/toolbar"
+        android:background="?attr/colorPrimary"
+        android:minHeight="?attr/actionBarSize"
+        android:fitsSystemWindows="true"
+        app:title="Sensor QuickView"
+        app:titleTextColor="@color/white"
+        app:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar"></android.support.v7.widget.Toolbar>
```
Comment (CloudyPadmal) [inline app/src/main/res/layout/sensor_main.xml]: `</android.support.v7.widget.Toolbar>` part is not necessary as you can close the tag with `app:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar" />`

Comment (CloudyPadmal) [reply app/src/main/res/layout/sensor_main.xml]: Please fix that in all the occurrences in this PR!

File: app/src/main/java/org/fossasia/pslab/activity/WavegenActivity.java [hunk 2]
```diff
@@ -4,20 +4,43 @@
...
+    AppBarLayout appBarLayout;
     @Override
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_wave_gen);
+        setContentView(R.layout.wave_gen_toolbar);
+
+        ButterKnife.bind(this);
+        setSupportActionBar(toolbar);
+        getSupportActionBar().setTitle("Wave Generator");
```
Comment (CloudyPadmal) [inline app/src/main/java/org/fossasia/pslab/activity/WavegenActivity.java]: Use string resources instead of hard-coded strings!

**Comment by CloudyPadmal** [comment]
Any update @bravoo84 ?

**Comment by bravoo84** [comment]
@CloudyPadmal I have texted you on hangouts!

**Comment by CloudyPadmal** [comment]
@bravoo84 I didn't receive any message. You can continue the conversation here!

**Comment by bravoo84** [comment]
@CloudyPadmal I am currently having University exams,so i am unable to make changes,but i will surely update the PR after 30th April

**Comment by CloudyPadmal** [comment]
@bravoo84 thank you for the PR! There are few more changes needed. The gradle files need to be removed from the commit.

**Comment by harsh-2711** [comment]
@bravoo84  Are u working on this PR or should I continue from here ?

**Comment by bravoo84** [comment]
Yh you continue @harsh-2711

**Comment by CloudyPadmal** [comment]
@harsh-2711 is working on the issue #770 and sent a new PR in #872 as author is okay with the changes.

**Pull request closed without merging**